.solhintignore

```diff
@@ -1 +1,2 @@
 node_modules/
+artifacts/
```

Comment:
Excluding artifacts. At compile time, solc can create folders ending in **.sol** that will match (and make fail) solhint contracts expansion filenames